### DISCUSSION TITLE
Where is the "gridfinity baseplate" option when installing via Fusion App store?

### DISCUSSION THREAD

**Discussion by ttyrawa** [Le0Michine/FusionGridfinityGenerator#124] [Q&A]
I have just installed version 1.4.1.0 for windows via the fusion app store from the link in the Readme. After installation I only see option for gridfinity Bin in the solid menu, there is no option for the baseplate. I have tried reinstalling but issue persists.

**Comment by rumlak** [comment]
![image](https://github.com/user-attachments/assets/de21c7fd-a3b1-4bb5-b7a0-1bb23de85cb4)
